Tests/FeaturevisorSDKTests/ConditionsTests.swift

```diff
@@ -103,6 +103,38 @@ class ConditionsTests: XCTestCase {
         XCTAssertTrue(conditionIsMatched(condition: condition, context: context))
     }
 
+    func testStringInArray() {
+        //Given
+        let condition = PlainCondition(
+            attribute: "text",
+            operator: .in,
+            value: .array(["test-1", "test-2"])
+        )
+        var context = Context()
+
+        //When
+        context["text"] = .string("test-1")
+
+        //Then
+        XCTAssertTrue(conditionIsMatched(condition: condition, context: context))
```

Comment:
👍

Reply:
it would add more confidence if we also have an assertion that fails.

Reply:
Yea, should add such test. I am gonna add it!